src/client/action/createRow.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CreateRow extends React.Component{
+
+  constructor(props){
+    super(props);
+    this.state = {
+      seq: '',
+      status: '',
+      category: '',
+      title: '',
+      owner: '',
+      priority: ''
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange(e){
+    const target = e.target;
+    const value = target.value;
+    const name = target.name;
+
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleSubmit(e){
+    const { seq, status, category, title, owner, priority } = this.state;
+    const { onSubmit } = this.props;
+    const formData = {
+      seq: seq,
+      status: status,
+      category: category,
+      title: title,
+      owner: owner,
+      priority: priority
+    };
+    e.preventDefault();
+    onSubmit(formData);
+  }
+
+  render(){
+    const { seq, status, category, title, owner, priority } = this.state;
+    const { isClicked } = this.props;
+    const reveal = isClicked ? 'block' : 'none';
+    return(
+      <div className="create__row" style={{display: reveal}}>
```

Comment:
Bad smell for high duplication. Funtion-nize for your `input` and `label` for `form` to improve readability.

```js
  const labelNameList = ['seq', 'status', 'category', 'title', 'owner', 'priority' ];

    return(
      <div className="create__row" style={{display: reveal}}>
        <form className="create__form" onSubmit={this.handleSubmit}>
          {
            labelNameList.map((index, name) => {
              return (
                <label htmlFor={name} key={`create-row-${name}`}>
                  {name}
                  <input
                    type="text"
                    name={name}
                    value={this.state[name]}
                    onChange={this.handleInputChange}
                    required
                  />
                </label>
              );
            })
          }
          <input type="submit" text="Submit" />
        </form>
      </div>
```

Reply:
Also, `seq` should be taken as kind of row `key` and I thought it could be editable.